src/WorkerPool_TEST.cc

```diff
@@ -92,7 +92,14 @@ TEST(WorkerPool, WaitWithTimeout)
       {
         workSentinel = 5;
       });
+#ifndef _WIN32
+# pragma GCC diagnostic push
+# pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#endif
   Time time(5);
```

Comment:
Instead of suppressing warnings, I think we should update this to use chrono. Otherwise, when the `common::Time` class is removed, how will this test work?

This means deprecating the `WaitForResults` function and adding a new one that takes chrono.